packages/workspace/src/schematics/shared-new/shared-new.ts

```diff
@@ -56,6 +56,9 @@ class RunPresetTask {
 
 function createPresetTaskExecutor(cli: string, opts: Schema) {
   const cliCommand = cli === 'angular' ? 'ng' : 'nx';
+  const interactive = !!process.argv.find(
```

Comment:
This doesn't find `--interactive false` does it? Is this consistent with what `create-nx-workspace` would read?